addons/bestja_account_deletion/wizards.py

```diff
@@ -23,7 +23,16 @@ def delete_account(self):
 class ReportWizard(models.TransientModel):
     _name = 'bestja.report_account_deletion_wizard'
 
-    reason = fields.Text(string="Dlaczego chcesz usunąć konto?", required=True)
+    REASONS = [
+        ('no_time', 'Nie mam czasu.'),
+        ('location_change', 'Zmieniam miejsce zamieszkania.'),
+        ('personal_change', 'Zmienia się moja sytuacja osobista (np. kończę studia).'),
+        ('bad_offers', 'Oferty nie spełniły moich oczekiwań.'),
+        ('no_satisfaction', 'Wolontariat nie sprawia mi już satysfakcji.'),
+        ('else', 'Inny (wpisz powód)'),
+    ] 
+    reason = fields.Selection(REASONS, string="Dlaczego chcesz usunąć konto?", required=True)
+    reason_description = fields.Char()
```

Comment:
This should be set in the `reason_for_deleting_account` field when `reason` is “else”.